influxdata/tokenizer.go

```diff
@@ -13,7 +13,25 @@ const (
 	minRead = minGrow / 2
 )
 
-// Tokenizer implements low level parsing of a line-protocol message.
+var (
+	measurementChars      = newByteSet(",").union(whitespace).invert()
+	measurementEscapes    = newEscaper(" ,")
+	tagKeyChars           = newByteSet(",=").union(whitespace).invert()
+	tagKeyEscapes         = newEscaper(",= ")
+	tagValChars           = newByteSet(",").union(whitespace).invert()
+	tagValEscapes         = newEscaper(", =")
+	fieldSeparatorSpace   = newByteSet(" \t\r\f")
+	fieldKeyChars         = tagKeyChars
+	fieldKeyEscapes       = tagKeyEscapes
+	fieldStringValChars   = newByteSet(`"`).invert()
```

Comment:
Single quotes are forbidden in field value strings: https://docs.influxdata.com/influxdb/v2.0/reference/syntax/line-protocol/#quotes

Reply:
The doc is misleading in this respect. See my comment elsewhere.